crates/diman_lib/Cargo.toml

```diff
@@ -9,4 +9,9 @@ description = "Commonly used types for internal use in diman."
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/tehforsch/diman"
 
+[features]
+std = []
+num-traits-libm = ["num-traits/libm"]
```

Comment:
I chose to make neither default. If you envision external users of `diman_lib`, perhaps we should have `default = ["std"]`.

Reply:
I don't really see anyone using diman_lib outside of diman, so I think your choice is fine.